component/src/main/java/org/wso2/extension/siddhi/execution/geo/internal/utils/Utilities.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.extension.siddhi.execution.geo.internal.utils;
+
+import java.util.regex.Pattern;
+
+/**
+ * This class provides validation method to validate IP addresses.
+ */
+public class Utilities {
+
+    private static final Pattern IPV4_PATTERN =
+            Pattern.compile(
+                    "^(25[0-5]|2[0-4]\\d|[0-1]?\\d?\\d)(\\.(25[0-5]|2[0-4]\\d|[0-1]?\\d?\\d)){3}$");
+
+    private static final Pattern IPV6_STD_PATTERN =
+            Pattern.compile(
+                    "^(?:[0-9a-fA-F]{1,4}:){7}[0-9a-fA-F]{1,4}$");
+
+    private static final Pattern IPV6_HEX_COMPRESSED_PATTERN =
+            Pattern.compile(
+                    "^((?:[0-9A-Fa-f]{1,4}(?::[0-9A-Fa-f]{1,4})*)?)::((?:[0-9A-Fa-f]{1,4}(?::[0-9A-Fa-f]{1,4})*)?)$");
+
+    /**
+     * Determine if the given string is a valid IPv4 or IPv6 address.  This method
+     * uses pattern matching to see if the given string could be a valid IP address.
+     *
+     * @param ipAddress A string that is to be examined to verify whether or not
+     *                  it could be a valid IP address.
+     * @return true if the string is a value that is a valid IP address,
+     * false otherwise.
+     */
```

Comment:
This comment can be formatted

Reply:
Fixed https://github.com/wso2-extensions/siddhi-execution-geo/pull/61/commits/a08d5e724a57f053d7d18ae441ace09bafeb208a